docs/overview/roadmap.md

```diff
@@ -6,55 +6,17 @@ description: Roadmap of Backstage
 
 ## The Backstage Roadmap
 
-Backstage is currently under rapid development. This page details the project’s
-public roadmap, the result of ongoing collaboration between the core maintainers
-and the broader Backstage community. Treat the roadmap as an ever-evolving guide
-to keep us aligned as a community on:
+Backstage is currently under rapid development. This page details the project’s public roadmap, the result of ongoing collaboration between the core maintainers and the broader Backstage community. 
 
-- Upcoming enhancements and benefits,
-- Planning contributions and support,
-- Planning the project’s adoption,
-- Understanding what things are coming soon,
-- Avoiding duplication of work
+The Backstage roadmap lays out both [“what’s next”](#whats-next) and [“future work”](#future-work). With "next" we mean features planned for release within the ongoing quarter from January through March 2022. With "future" we mean features in the radar, but not yet scheduled.
 
-### How to influence the roadmap
+| What's next | [Future work](#future-work) |
```

Comment:
Feeling that "What's next" should get a link here too 😅

Reply:
Yep, added!
Thanks